src/main/java/uk/ac/ebi/eva/metrics/metric/ClusteringMetric.java

```diff
@@ -0,0 +1,49 @@
+package uk.ac.ebi.eva.metrics.metric;
+
+public enum ClusteringMetric implements Metric {
```

Comment:
`AccessioningMetric`, `AccessioningMetricComputeImpl`, `ClusteringMetric`, `ClusteringMetricComputeImpl`
Are these class meant to be in the generic metric library ?
These are unlikely to be used by more than one repo

Reply:
Make sense, I was trying to look at it from the eye of one stop all library for metrics. But if we push the more specific classes to their repos, I guess it will make it easy to create new metrics for them later without touching this repo